quad_utils/rviz/example_with_terrain.rviz

```diff
@@ -3,9 +3,11 @@ Panels:
     Help Height: 0
```

Comment:
Revert this file for now since we aren't ready to fully support the state estimate yet.